lib/adapter.js

```diff
@@ -155,6 +162,7 @@ class MochaAdapter {
         if (err) {
             this.runner.lastError = err
         }
+        process.emit(event, message)
```

Comment:
See comment [here](https://github.com/webdriverio/webdriverio/pull/896/files#r47439176). We need to make sure that adapter work across frameworks. We should be careful with merging webdriverio related data with framework data